packages/components/src/Drawer/Drawer.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { cleanup } from "@testing-library/react";
+import { Drawer } from ".";
+
+afterEach(cleanup);
+
+it("renders a Drawer", () => {
```

Comment:
Would probably be good to test that it doesn't render a drawer if `open` is false.

Reply:
Also, if that `onRequestClose` is called when the `✖️` is clicked.